code/parse/lua.cpp

```diff
@@ -1493,7 +1493,7 @@ ADE_FUNC(isValid, l_Font, NULL, "True if valid, false or nil if not", "boolean",
 	if(!ade_get_args(L, "o", l_Font.GetPtr(&fh)))
 		return ADE_RETURN_NIL;
 
-	return ade_set_args(L, "b", fh->isValid());
+	return ade_set_args(L, "b", fh != nullptr && fh->isValid());
```

Comment:
Even if the `l_Font.GetPtr(&fh)` fails to set `fh` to anything, this won't help defend against it because line 1492 doesn't initialize it, so it would be uninitialized memory rather than a null pointer.